src/components/elements/lottie.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import ReactLottie from 'react-lottie'
+import PropTypes from 'prop-types'
+
+export const Lottie = ({
+    animationData,
+    autoplay = true,
+    height = 40,
+    loop = true,
+    onAnimationComplete,
+    width = 40,
+}) => {
+    const default_options = {
+        loop,
+        autoplay,
+        animationData,
+        rendererSettings: {
+            preserveAspectRatio: 'xMidYMid slice',
+        },
+    }
+
+    const eventListeners = []
+    if (onAnimationComplete) {
+        eventListeners.push({ eventName: 'complete', callback: onAnimationComplete })
+    }
+
+    return (
```

Comment:
can we add example in the implementation-guide.md, and add this https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/preserveAspectRatio in it 🙏